### PR TITLE
add release step to enable AIP-32

### PR DESCRIPTION
**Pull request by msmouse**
### Description
enable storage refund feature flags.

requires module events https://github.com/aptos-labs/aptos-core/pull/9811/commits/80afd5dd525a357ebc648150c84202ebaf8bbcc7

### Test Plan

framework upgrade test

### CODE REVIEW

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -30,3 +30,15 @@ proposals:
           transaction_shuffler_type:
             deprecated_sender_aware_v1: 32
           block_gas_limit : 35000
+  - name: enable_enable_storage_deletion_refund
+    metadata:
+      title: "Enable Storage Deletion Refund"
+      description: "AIP-32: Storage Deletion Refund"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/127"
+    execution_mode: SingleStep
```
Comment (movekevin) [inline aptos-move/aptos-release-builder/data/release.yaml:38]: Multistep please

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml:38]: So what does SingleStep mean? (The update_sequence has only one step?)

Comment (movekevin) [reply aptos-move/aptos-release-builder/data/release.yaml:38]: No it uses a different cli and smart contract code path thats no longer preferred. Multistep works for when theres a single step

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml:38]: Gotcha. If that's the case we should mark SingleStep deprecated?

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml]: cc @sitalkedia  @lightmark  sorry

**Review by grao1991 — APPROVED** [review]

**Review by zekun000 — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 2]
```diff
@@ -30,3 +30,15 @@ proposals:
...
           block_gas_limit : 35000
+  - name: enable_enable_storage_deletion_refund
+    metadata:
+      title: "Enable Storage Deletion Refund"
+      description: "AIP-32: Storage Deletion Refund"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/127"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
```
Comment (movekevin) [inline aptos-move/aptos-release-builder/data/release.yaml:41]: Is there any initialize function that needs to be called? I thought I saw one

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml:41]: ugh....

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml:41]: So it needs to be a `RawScript`?

Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml:64]: updated

**Review by movekevin — APPROVED** [review]

**Pull request merged**